src/pages/workspace/accounting/intacct/import/SageIntacctToggleMappingsPage.tsx

```diff
@@ -50,11 +52,23 @@ function SageIntacctToggleMappingsPage({route}: SageIntacctToggleMappingsPagePro
 
     const policy = usePolicy(route.params.policyID);
     const mappingName: SageIntacctMappingName = route.params.mapping;
-    const policyID: string = policy?.id ?? '-1';
-
+    const policyID: string = policy?.id ?? CONST.DEFAULT_POLICY_ID;
```

Comment:
Please read through `contributingGuides/STYLE.md` (Default value for inexistent IDs). I think we shouldn't add `DEFAULT_POLICY_ID` and instead use no id at all, what do you think? @sumo-slonik

Reply:
Yeah, let's avoid any defaulting values for string ids!
In this case maybe you can try to use `route.params.policyID` - this way you won't need to handle undefined cases, wdyt?

Reply:
It seems to me that the example in `contributingGuides/STYLE.md`  indicates that we should have no value, but I wanted to keep the sense of the code. But I will do it as you suggest.

Reply:
Yes please lets remove this